### PR TITLE
Add full website functionality

### PR DESCRIPTION
**Pull request by tongoonamujera**
- [x] Add linters
- [x] Add delete method
- [x] Add clear method
- [x] fix linter errors
- [x] Add more styles

### CODE REVIEW

File: src/htmlDisplay.js [hunk 1]
```diff
@@ -8,25 +8,19 @@ export const createTodos = (todoList) => {
...
-        <section><i class='fas fa-ellipsis-v'></i></section>
+      <div class='d-todo' draggable="true">
+      ${ task.completed != true ? `<input type="checkbox" value="${index}" />` : `<input type="checkbox" value="${index}" checked />` }
+        <p contentEditable="true" data-id="${index}">${task.description}</p>
+        <section><i class='far fa-trash-alt fa-2x' data-id="${index}"></i></section>
       </div>
     `;
+
+    todoList.appendChild(todoDiv);
   });
```
Comment (henatan99) [inline src/htmlDisplay.js]: - [x] For some reason the DOM is not displaying the first TODO item. Kindly fix this bug. :pray: 

Screenshot from 2021-12-03 12-51-33

File: src/index.js [hunk 2]
```diff
@@ -3,10 +3,46 @@ import './fontawesome/css/all.css';
...
+  const task = new TaskStorage();
+  if (e.keyCode == 13){
+    task.editTask(e);
+    location.reload();
+  }
+});
+
+todo.addEventListener('click', (e) => {
+  const task = new TaskStorage();
+  task.deleteTask(e);
```
Comment (henatan99) [inline src/index.js:41]: - [x] When I delete a to-do item, it automatically updates the localStorage but it doesn't update the DOM unless page is refreshed. Kindly fix this. :pray: 

Screenshot from 2021-12-03 12-55-30

Screenshot from 2021-12-03 12-55-36

File: src/index.js [hunk 3]
```diff
@@ -3,10 +3,46 @@ import './fontawesome/css/all.css';
...
 
-const todo = document.querySelector('.todo');
+/* eslint-disable */
+document.querySelector('.form-btn').addEventListener('click', (e) => {
+  const textField = document.querySelector('input[type="text"]');
+  const tasks = {
+  description: textField.value,
+  completed: false,
+  index: 0,
+  }
```
Comment (henatan99) [inline src/index.js:13]: - [x] This expression assigns the index value of a newly created Todo item to be zero. I think it should be one plus the last element index. Kindly fix this. :pray: 

Screenshot from 2021-12-03 12-59-52

File: src/localStorage.js [hunk 4]
```diff
@@ -44,9 +30,50 @@ export class TaskStorage {
...
+
+  removeTasks = () => {
+    const allTasks = this.getTask();
+    const tasks = allTasks.filter((task) => {
+      return task.completed != true;
+    });
+
+    console.log(tasksi);
+    localStorage.setItem('alltasks',JSON.stringify(tasks));
+  }
```
Comment (henatan99) [inline src/localStorage.js:53]: - [x] When a todo item is deleted the indexes of the remaining todos should be re-adjusted to be consecutive integers. For example if the initial indexes were 1, 2, 3, 4 and If todo item 3 is deleted the new todos should have indexes of 1, 2, 3 not 1, 2, 4. Kindly fix this. :pray: 

Screenshot from 2021-12-03 13-06-17

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @tongoonamujera , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**